scripts/issues/2595/monitor.py

```diff
@@ -5,7 +5,7 @@
 import csv
 import os
 from shutil import move
-base_url = 'http://localhost:4848/monitoring/domain/server/resources/dvnDbPool'
+base_url = 'http://localhost:4848/monitoring/domain/server/resources/dvnDbPool1'
```

Comment:
Did you rename the existing pool?  We probably don't want to do that.

Reply:
Fixed.